lib/ui/octicon.jsx

```diff
@@ -64,7 +64,7 @@ export class Octicon {
 			classNames.push('disabled');
 		}
 
-		const fill = disabled ? '#505868' : '#9da5b4';
+		const fill = 'var(--standard-text-color)';
```

Comment:
I'm not too sure on the change of colour here as it makes it different to everything else on the toolbar

Reply:
It is the same color the icons on the right have

Reply:
Maybe there's a difference here on mac? For me the icon is black 
![image](https://user-images.githubusercontent.com/8705251/140296110-ce2e1e89-4b23-49a0-b732-3c16384798e8.png)
Whereas the other icons are grey 
![image](https://user-images.githubusercontent.com/8705251/140296172-7101e85a-d5ca-435a-b4a3-baf1982db695.png)

Reply:
oh interesting. I'll check that. Perhaps its system dark/light mode that is making an issue here.

Reply:
looks like that variable was set by a different package and is not available on OSX.
The thing is that all buttons use `color:inherit` and get the color from the atom-panel. But there isn't a fill color. So now I use `getComputedStyle(document.querySelector("atom-panel")).color;`. 
Only "problem" it is not updated when you switch dark/light themes. You have to click it again in order to get the right color